.github/workflows/cicd.yml

```diff
@@ -22,4 +22,4 @@ jobs:
         python-version: 3.8
     - name: Run tests
       run: |
-        pip install . && python setup.py pytest
\ No newline at end of file
+        pip install numpy && pip install . && python setup.py pytest
```

Comment:
I think you should use tox ;-)
https://github.com/cogeotiff/rio-tiler/blob/master/tox.ini#L8-L9

Reply:
probably lol